packages/Badge/src/Badge.spec.tsx

```diff
@@ -9,13 +9,22 @@ describe('packages/Badge', () => {
   const className = 'test-pill-class';
   const child = 'Bubble Pill';
 
+  function isElement(node: Node): node is Element {
```

Comment:
We should probably put these type of functions inside `lib` package? Maybe make a ticket to "Move all type utils to  `@leafygreen-ui/lib/types`"?

Reply:
Works for me

Reply:
So we want a separate ticket, as opposed to doing it as part of this one, then?

Reply:
Up to you.

Reply:
Filed [PD-313](https://jira.mongodb.org/browse/PD-313)